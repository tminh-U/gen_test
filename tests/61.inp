13
15 6 18 2 15 17 9 13 11 13 15 18 10 
12
3 3
10 12
2 9
11 12
1 6
1 8
5 9
11 13
3 7
13 13
4 11
13 13
