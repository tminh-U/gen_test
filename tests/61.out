2
8
29
4
19
28
9
5
11
2
56
2
